src/target/nrf51.c

```diff
@@ -273,6 +278,30 @@ static bool nrf51_cmd_erase_uicr(target *t, int argc, const char **argv)
 	return true;
 }
 
+static bool nrf51_cmd_protect_flash(target *t, int argc, const char **argv)
+{
+	(void)argc;
+	(void)argv;
+	tc_printf(t, "protect flash..\n");
+
+	/* Enable write */
+	target_mem_write32(t, NRF51_NVMC_CONFIG, NRF51_NVMC_CONFIG_WEN);
+
+	/* Poll for NVMC_READY */
+	while (target_mem_read32(t, NRF51_NVMC_READY) == 0)
```

Comment:
While this code is correct, please include the outer {}'s for the while loop for readability. As written this is bug-prone and confusing to newcomers.

Reply:
Thanks @dragonmux! Added the braces for the two while loops.